apps/api-documenter/src/documenters/YamlDocumenter.ts

```diff
@@ -668,4 +707,42 @@ export class YamlDocumenter {
     console.log('Deleting old output from ' + this._outputFolder);
     FileSystem.ensureEmptyFolder(this._outputFolder);
   }
+
+  // Parses the tocConfig object to build a pointers map of nodes where we want to sort out the API items
+  // tslint:disable-next-line: no-any
+  private _generateTocPointersMap(tocConfig: any): void {
+    for (const tocItem of tocConfig.items) {
+      if (tocItem.items) {
+        if (tocItem.items.length > 0) {
```

Comment:
Instead of nesting these if statements, let's do 
```tsx
if (tocItem.items && tocItem.items.length > 0) {
```

Reply:
Thanks... I do vaguely remember that I stopped here for a bit and had a reason why I nested them like this but it doesn't look I remember why and removing the nesting makes sense.